dpdata.py

```diff
@@ -124,6 +134,27 @@ def scrub_data(self):
         self.__fixup_nomail_flag()
         self.__copy_spouse_email()
 
+    def calculate_homeschool(self, student_list):
+        '''given a list of students for the donor, calculate their homeschool.
+        Return the elementary school for all students unless all are in BIS.
+        If more than one elementary school, return ''.
+        '''
+        current_school = ''
+        bis=False
+        for student in student_list:
+            if student['SCHOOL']  in ['FRANKLIN', 'HOOVER', 'LINCOLN','MCKINLEY','ROOSEVELT', 'WASHINGTON']:                
```

Comment:
Simpler to check `if student['SCHOOL'] != 'BIS'`?

Reply:
I don't want to include if they are alumns so it can't just be a check for not BIS.  I guess I could have check to see that it's not BIS nor ALUM nor NOBSD.  it would be a shorter list -- but  i think being concise about what i'm checking for is probably better in this case.